contracts/SimpleAuction.sol

```diff
@@ -86,7 +85,8 @@ contract SimpleAuction is /* AccessControl, */ MetaTxContext, ERC1155HolderUpgra
         address indexed seller,
         uint256 indexed nftId,
         uint256 price,
-        uint256 closeTimeSec);
+        uint256 closeTimeSec,
+        bool secured);
```

Comment:
@AlekseiMisyukevich Also please make sure to coordinate the deployment of this event change with the Freeport event listener (with @spoluyan).

Reply:
sure, will do.